index.js

```diff
@@ -75,27 +75,24 @@ function gulpRequireTasks (options) {
      */
     function taskNameFromPath (modulePath) {
       const relativePath = path.relative(options.path, modulePath);
-      return removeExtension(relativePath)
-        .split(path.sep).join(options.separator)
-      ;
+      const pathInfos = path.parse(relativePath);
+      var taskNameParts = []
```

Comment:
`taskNameParts` is a reference to array, and you are changing only content of the array, not the reference itself, so it's perfectly fine to use `const`.